.github/workflows/ci.yml

```diff
@@ -32,3 +32,4 @@ jobs:
     secrets: inherit
     with:
       runner_image_name: ${{ needs.build-push-docker-image.outputs.runner_image_name }}
+      deploy_localstack: true
```

Comment:
Adding localstack deploy check for every CI can slow down the development process. Can we add localstack deploy check for stage and PR only?

Reply:
yeah please, not sure how to do that though

Reply:
Ok vai, I'll update it